source/_components/sensor.yr.markdown

```diff
@@ -68,6 +68,21 @@ monitored_conditions:
       description: Medium cloud level.
     highClouds:
       description: High cloud level.
+latitude:
+  description: Manually specify latitude.
+  required: false
+  type: number
```

Comment:
Is number a legit type? Or should it be a float?

Reply:
Is not a float a sub type of number?

Reply:
Well I see [here](https://developers.home-assistant.io/docs/en/documentation_create_page.html) number not as variable type.

Reply:
In the docs that would be translated into a `float`.
`number` is not valid in the docs.